lefthook.yaml

```diff
@@ -42,6 +42,91 @@ pre-commit:
       glob: "test/**/*.{test,spec}.{ts,tsx}"
       run: pnpm run check_mock_isolation {staged_files}
 
+    10.1_check_biome_disable:
+      glob: "**/*.{ts,tsx}"
+      run: |
+        set -e
+        if [ -f .venv/bin/python3 ]; then
+          .venv/bin/python3 .github/workflows/scripts/biome_disable_check.py --files {staged_files}
+        elif [ -f .venv/Scripts/python.exe ]; then
+          .venv/Scripts/python.exe .github/workflows/scripts/biome_disable_check.py --files {staged_files}
+        else
+          echo "Error: Python virtual environment not found."
```

Comment:
Explain that they may also need to install the python3-venv and/or python-venv packages